src/Angor/Client/wwwroot/index.html

```diff
@@ -6,46 +6,50 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />
     <title>Angor</title>
     <base href="/" />
-    <link href="css/bootstrap/bootstrap.min.css" rel="stylesheet" />
-    <link href="css/bootstrap/bootstrap.theme.min.css" rel="stylesheet" />
-    <link href="css/app.css" rel="stylesheet" />
-    <link rel="icon" type="image/svg+xml" href="angor-icon.svg" />
-    <link rel="icon" type="image/png" href="favicon.png" />
-    <link href="Angor.Client.styles.css" rel="stylesheet" />
-    <link href="manifest.json" rel="manifest" />
-    <link rel="apple-touch-icon" sizes="512x512" href="icon-512.png" />
-    <link rel="apple-touch-icon" sizes="192x192" href="icon-192.png" />
-
+    <link rel="icon" type="image/svg+xml" href="/assets/img/angor-icon.svg" />
+    <link rel="icon" type="image/png" href="/assets/img/favicon.png" />
 
+    <link href="manifest.json" rel="manifest" />
+    <link rel="apple-touch-icon" sizes="512x512" href="/assets/img/icon-512.png" />
+    <link rel="apple-touch-icon" sizes="192x192" href="/assets/img/icon-192.png" />
+    <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700,900|Roboto+Slab:400,700" />
+    <link href="/assets/css/nucleo-icons.css" rel="stylesheet" />
+    <link href="/assets/css/nucleo-svg.css" rel="stylesheet" />
+    <script src="https://kit.fontawesome.com/42d5adcbca.js" crossorigin="anonymous"></script>
+    <link href="Angor.Client.styles.css" rel="stylesheet" />
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons+Round" rel="stylesheet" />
```

Comment:
We are adding here calls to google api, it is common practice but perhaps we can try to find a way to avoid that?

Reply:
ok perfect I will create a task to remind us of this